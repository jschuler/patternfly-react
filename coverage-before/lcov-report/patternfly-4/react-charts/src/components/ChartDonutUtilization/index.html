<!doctype html>
<html lang="en">
<head>
    <title>Code coverage report for patternfly-4/react-charts/src/components/ChartDonutUtilization</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="../../../../../prettify.css" />
    <link rel="stylesheet" href="../../../../../base.css" />
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(../../../../../sort-arrow-sprite.png);
        }
    </style>
</head>
<body>
<div class='wrapper'>
  <div class='pad1'>
    <h1>
      <a href="../../../../../index.html">All files</a> patternfly-4/react-charts/src/components/ChartDonutUtilization
    </h1>
    <div class='clearfix'>
      <div class='fl pad1y space-right2'>
        <span class="strong">86.54% </span>
        <span class="quiet">Statements</span>
        <span class='fraction'>135/156</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">57.14% </span>
        <span class="quiet">Branches</span>
        <span class='fraction'>56/98</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">86.96% </span>
        <span class="quiet">Functions</span>
        <span class='fraction'>20/23</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">88.51% </span>
        <span class="quiet">Lines</span>
        <span class='fraction'>131/148</span>
      </div>
    </div>
    <p class="quiet">
      Press <em>n</em> or <em>j</em> to go to the next uncovered block, <em>b</em>, <em>p</em> or <em>k</em> for the previous block.
    </p>
  </div>
  <div class='status-line high'></div>
<div class="pad1">
<table class="coverage-summary">
<thead>
<tr>
   <th data-col="file" data-fmt="html" data-html="true" class="file">File</th>
   <th data-col="pic" data-type="number" data-fmt="html" data-html="true" class="pic"></th>
   <th data-col="statements" data-type="number" data-fmt="pct" class="pct">Statements</th>
   <th data-col="statements_raw" data-type="number" data-fmt="html" class="abs"></th>
   <th data-col="branches" data-type="number" data-fmt="pct" class="pct">Branches</th>
   <th data-col="branches_raw" data-type="number" data-fmt="html" class="abs"></th>
   <th data-col="functions" data-type="number" data-fmt="pct" class="pct">Functions</th>
   <th data-col="functions_raw" data-type="number" data-fmt="html" class="abs"></th>
   <th data-col="lines" data-type="number" data-fmt="pct" class="pct">Lines</th>
   <th data-col="lines_raw" data-type="number" data-fmt="html" class="abs"></th>
</tr>
</thead>
<tbody><tr>
	<td class="file high" data-value="ChartDonutThreshold.tsx"><a href="ChartDonutThreshold.tsx.html">ChartDonutThreshold.tsx</a></td>
	<td data-value="88.73" class="pic high"><div class="chart"><div class="cover-fill" style="width: 88%;"></div><div class="cover-empty" style="width:12%;"></div></div></td>
	<td data-value="88.73" class="pct high">88.73%</td>
	<td data-value="71" class="abs high">63/71</td>
	<td data-value="56.25" class="pct medium">56.25%</td>
	<td data-value="48" class="abs medium">27/48</td>
	<td data-value="90.91" class="pct high">90.91%</td>
	<td data-value="11" class="abs high">10/11</td>
	<td data-value="90.91" class="pct high">90.91%</td>
	<td data-value="66" class="abs high">60/66</td>
	</tr>

<tr>
	<td class="file high" data-value="ChartDonutUtilization.tsx"><a href="ChartDonutUtilization.tsx.html">ChartDonutUtilization.tsx</a></td>
	<td data-value="84.71" class="pic high"><div class="chart"><div class="cover-fill" style="width: 84%;"></div><div class="cover-empty" style="width:16%;"></div></div></td>
	<td data-value="84.71" class="pct high">84.71%</td>
	<td data-value="85" class="abs high">72/85</td>
	<td data-value="58" class="pct medium">58%</td>
	<td data-value="50" class="abs medium">29/50</td>
	<td data-value="83.33" class="pct high">83.33%</td>
	<td data-value="12" class="abs high">10/12</td>
	<td data-value="86.59" class="pct high">86.59%</td>
	<td data-value="82" class="abs high">71/82</td>
	</tr>

</tbody>
</table>
</div><div class='push'></div><!-- for sticky footer -->
</div><!-- /wrapper -->
<div class='footer quiet pad2 space-top1 center small'>
  Code coverage
  generated by <a href="https://istanbul.js.org/" target="_blank">istanbul</a> at Fri Jan 17 2020 19:28:45 GMT-0500 (Eastern Standard Time)
</div>
</div>
<script src="../../../../../prettify.js"></script>
<script>
window.onload = function () {
        if (typeof prettyPrint === 'function') {
            prettyPrint();
        }
};
</script>
<script src="../../../../../sorter.js"></script>
<script src="../../../../../block-navigation.js"></script>
</body>
</html>
