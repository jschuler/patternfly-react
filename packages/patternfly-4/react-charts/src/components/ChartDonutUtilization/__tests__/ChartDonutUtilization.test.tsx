// Auto-generated do not edit

import * as React from 'react';
import { shallow } from 'enzyme';
import { ChartDonutUtilization } from '../ChartDonutUtilization';
import { ChartContainer } from '../../ChartContainer';

describe('ChartDonutUtilization test', () => {
  it('ChartDonutUtilization should match snapshot', () => {
    const view = shallow(
      <ChartDonutUtilization
        allowTooltip={true}
				animate={undefined}
				ariaDesc={undefined}
				ariaTitle={undefined}
				capHeight={undefined}
				categories={undefined}
				colorScale={undefined}
				constrainToVisibleArea={undefined}
				containerComponent={<ChartContainer />}
				cornerRadius={undefined}
				data={undefined}
				dataComponent={undefined}
				desc={undefined}
				endAngle={undefined}
				eventKey={undefined}
				events={undefined}
				externalEventMutations={undefined}
				groupComponent={undefined}
				height={42}
				innerRadius={undefined}
				invert={false}
				legendAllowWrap={undefined}
				labelComponent={undefined}
				labelPosition={undefined}
				legendComponent={undefined}
				legendData={undefined}
				legendOrientation={undefined}
				legendPosition={'bottom'}
				labelRadius={undefined}
				labels={undefined}
				name={undefined}
				origin={undefined}
				padAngle={undefined}
				padding={undefined}
				radius={undefined}
				sharedEvents={undefined}
				showStatic={true}
				sortKey={undefined}
				sortOrder={undefined}
				standalone={true}
				startAngle={undefined}
				style={undefined}
				subTitle={undefined}
				subTitleComponent={undefined}
				subTitlePosition={undefined}
				theme={{}/*unrecognizedType ChartThemeDefinition undefined*/}
				themeColor={undefined}
				themeVariant={undefined}
				title={undefined}
				thresholds={undefined}
				width={42}
				x={undefined}
				y={undefined}
      />);
    expect(view).toMatchSnapshot();
  });
});
