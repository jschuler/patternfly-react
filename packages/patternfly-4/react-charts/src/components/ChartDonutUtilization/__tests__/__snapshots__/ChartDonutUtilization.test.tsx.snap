// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ChartDonutUtilization test ChartDonutUtilization should match snapshot 1`] = `
<Fragment>
  <Component
    height={42}
    theme={Object {}}
    width={42}
  >
    <Component
      allowTooltip={true}
      data={
        Array [
          Object {
            "x": 0,
            "y": Object {},
          },
          Object {
            "y": 100,
          },
        ]
      }
      height={42}
      key="pf-chart-donut-utilization"
      legendPosition="bottom"
      standalone={false}
      theme={Object {}}
      width={42}
    />
  </Component>
</Fragment>
`;
